configs/desktop_savant_events_schemas.json

```diff
@@ -0,0 +1,1122 @@
+            "meta-cat-1": "'feature'",
+            "meta-cat-2": "'pwmgr'",
+            "flow-id": "extra.flow_id"
+          },
+          "schema": {
+            "$schema": "http://json-schema.org/schema#",
+            "type": "object",
+            "properties": {
+              "timestamp": {
+                "type": "number",
+                "minimum": 0
+              },
+              "category": {
+                "type": "string",
+                "enum": [
+                  "savant"
+                ]
+              },
+              "method": {
+                "type": "string",
+                "enum": [
+                  "pwmgr_use"
+                ]
+              }
+            },
+            "required": [
+              "timestamp",
+              "category",
+              "method"
+            ]
+          }
+        },
+        {
+          "name": "asked add pw",
+          "description": "browser asks client if they want to save or update a password used",
+          "amplitudeProperties": {
+            "meta-cat-1": "'popup'",
+            "meta-cat-2": "'pwmgr'",
+            "add-type": "value",
+            "flow-id": "extra.flow_id"
+          },
+          "schema": {
+            "$schema": "http://json-schema.org/schema#",
+            "type": "object",
+            "properties": {
+              "timestamp": {
+                "type": "number",
+                "minimum": 0
+              },
+              "category": {
+                "type": "string",
+                "enum": [
+                  "savant"
+                ]
+              },
+              "method": {
+                "type": "string",
+                "enum": [
+                  "pwmgr"
+                ]
+              },
+              "value": {
+                "type": "string",
+                "enum": [
+                  "ask"
+                ]
+              }
+            },
+            "required": [
+              "timestamp",
+              "category",
+              "method",
+              "value"
+            ]
+          }
+        },
+        {
+          "name": "saved pw",
+          "description": "client saves a password after being prompted",
+          "amplitudeProperties": {
+            "meta-cat-1": "'feature'",
+            "meta-cat-2": "'pwmgr'",
+            "flow-id": "extra.flow_id"
+          },
+          "schema": {
+            "$schema": "http://json-schema.org/schema#",
+            "type": "object",
+            "properties": {
+              "timestamp": {
+                "type": "number",
+                "minimum": 0
+              },
+              "category": {
+                "type": "string",
+                "enum": [
+                  "savant"
+                ]
+              },
+              "method": {
+                "type": "string",
+                "enum": [
+                  "pwmgr"
+                ]
+              },
+              "value": {
```

Comment:
[Events.yaml](https://dxr.mozilla.org/mozilla-central/source/toolkit/components/telemetry/Events.yaml#246) suggests this, and the other pwmgr events should be looking at "object" instead of "value"

Reply:
I'll ask Su to confirm.

Reply:
The `description`s for those events make reference to what will be populated in the "value field". I don't understand why both object and value exist with similar settings.

Reply:
@sunahsuh , correct, good catch, that's a typo on my part

@jklukas , good point. the difference is when client is asked if they want to save/update a password, we record object as ask, and depending on what was asked, put save or update in the value. when the client chooses to save or update, the event becomes object=save or update, with a blank value. 

definitely worth redesigning in the future